docs/data/data-grid/column-definition/column-definition.md

```diff
@@ -302,26 +302,9 @@ However, some types require additional properties to be set to make them work co
 
 {{"demo": "ColumnTypesGrid.js", "bg": "inline"}}
 
-## Custom column types
+### Custom column types
 
-You can extend the native column types with your own by simply spreading the necessary properties.
```

Comment:
Moved to https://deploy-preview-9228--material-ui-x.netlify.app/x/react-data-grid/custom-columns/